docs/index.md

```diff
@@ -1,41 +1,37 @@
+We aim to provide good quality, HTTP related packages to the PHP community. Our main product is
+[HTTPlug](http://httplug.io), but we also have other smaller packages to make using HTTP more convenient.
 
 
-## Getting started
+## HTTPlug
 
-Read our [tutorial](tutorial.md).
+[PSR-7](http://www.php-fig.org/psr/psr-7/) defines interfaces for HTTP requests and responses.
+However, it does not define how to create a request or how to send a request.
+HTTPlug abstracts from HTTP clients written in PHP, offering a simple interface.
+It also provides an implementation-independent plugin system to build pipelines regardless of the
+HTTP client implementation used.
 
+HTTPlug allows you to write reusable libraries and applications that need
+an HTTP client without binding to a specific implementation.
+When all packages used in an application only specify HTTPlug,
+the application developers can chose the client that fits best for their
+project and use the same client with all packages.
 
-## Overview
+There are clients implementing one of the HTTPlug interfaces directly,
+and adapter packages that implement the interface and forward the calls to HTTP clients not implementing the interface.
 
-PHP-HTTP is separated into several packages:
 
-- [Httplug](httplug.md), the HTTP client abstraction to send PSR-7 requests without binding to a specific implementation;
-- [Message Factory](message-factory.md) to create PSR-7 requests without binding to a specific implementation; 
-- [Plugins](plugins.md) to build client-agnostic pipelines for HTTP communication;
-- [Utilities](utils.md) convenience tools to simplify working with Httplug in your applications.
+Read our [tutorial](httplug/tutorial.md).
 
-See [package overview](package-overview.md) for a complete list.
 
-### Framework Integration
+# PHP-HTTP Components
 
-Httplug can be used in any PHP based project. Nonetheless, we provide particular integration for some popular frameworks:
-
-- [HttplugBundle](https://github.com/php-http/HttplugBundle/) integration with the Symfony framework.
-
-
-## History
-
-This project has been started by [Eric Geloen](https://github.com/egeloen) as [Ivory Http Adapter](https://github.com/egeloen/ivory-http-adapter). It never made it to a stable release, but it relied on PSR-7 which was not stable either that time. Because of the constantly changing PSR-7, Eric had to rewrite the library over and over again (at least the message handling part, which in most cases affected every adapter as well).
-
-In 2015, a decision has been made to move the library to it's own organization, so PHP HTTP was born.
-
-See [migrating](migrating.md) for a guide how to migrate your code from the Ivory adapter to Httplug.
+As explained above, PHP HTTP has room for anything that is connected to HTTP and PHP.
+Although [HTTPlug](http://httplug.io) is considered to be our "main product", we have other components as well.
```

Comment:
I think the navigation is enough for now, we can add a list later if we feel the need. However we have to maintain that list which we probably don't want to.